<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="left_layout">
      <item>
       <widget class="MeshViewerWidget" name="viewer">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>999999</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
        <widget class="QCheckBox" name="cbox_light_fixed">
         <property name="toolTip">
          <string>When checked, the light will stay steady into the World.
If not, the light will follow the rotation of the view.</string>
         </property>
         <property name="text">
          <string>Fixed Light</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbox_light_enable">
         <property name="text">
          <string>Enable Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbox_cull_bfaces">
         <property name="text">
          <string>Cull Back-Faces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_run_sequence">
         <property name="text">
          <string>Run Sequence</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLCDNumber" name="fps">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>FPS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="action_load_mesh"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuParameters">
    <property name="title">
     <string>Parameters</string>
    </property>
    <addaction name="actionColors"/>
    <addaction name="actionLight"/>
    <addaction name="actionSequence"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParameters"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_OBJ">
   <property name="text">
    <string>&amp;Load OBJ</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionLaplace_Beltrami">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Laplace-Beltrami</string>
   </property>
  </action>
  <action name="action_smooth">
   <property name="text">
    <string>&amp;Active</string>
   </property>
  </action>
  <action name="action_flat">
   <property name="text">
    <string>&amp;Flat</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_wireframe">
   <property name="text">
    <string>On/Off</string>
   </property>
  </action>
  <action name="action_load_mesh">
   <property name="text">
    <string>Import Mesh</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionSequence">
   <property name="text">
    <string>Sequence</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeshViewerWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>include/meshviewerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
